bika/lims/content/abstractanalysis.py

```diff
@@ -1176,7 +1176,8 @@ def getRetestOfUID(self):
     def getRetest(self):
         """Returns the retest that comes from this analysis, if any
         """
-        back_refs = get_backreferences(self, 'AnalysisRetestOf')
+        relationship = "{}RetestOf".format(self.portal_type)
```

Comment:
Is `get_backreferences` case insensitive? Because this returns smth. like `analysisRetestOf` instead of `AnalysisRetestOf`

Reply:
If no specific relationship has been set, at a first glance seems that is case sensitive: https://github.com/senaite/senaite.core/blob/master/bika/lims/browser/fields/uidreferencefield.py#L47
But honestly, haven't gone into that detail yet